src/wrapper/Makefile.am

```diff
@@ -1,17 +1,23 @@
 lib_LTLIBRARIES = dyad_wrapper.la
 dyad_wrapper_la_SOURCES = wrapper.c
 dyad_wrapper_la_LDFLAGS = \
+	-Wl,-rpath,'$(UCX_LIBDIR)' \
 	$(AM_LDFLAGS) \
 	-module \
 	-avoid-version \
 	-no-undefined \
 	-shared \
-	-export-symbols wrapper.sym
+	# -export-symbols wrapper.sym
```

Comment:
comments have problems in Makefile.am i would recommend removing this line is not needed. Make sure you remove the \ in previous line as wel.

Reply:
Thanks for catching this. I'll fix it.

Reply:
Done